include/ceed/ceed.h

```diff
@@ -382,13 +383,18 @@ CEED_EXTERN int CeedBasisGetInterp1D(CeedBasis basis, const CeedScalar **interp_
 CEED_EXTERN int CeedBasisGetGrad(CeedBasis basis, const CeedScalar **grad);
 CEED_EXTERN int CeedBasisGetGrad1D(CeedBasis basis, const CeedScalar **grad_1d);
 CEED_EXTERN int CeedBasisGetDiv(CeedBasis basis, const CeedScalar **div);
+CEED_EXTERN int CeedBasisGetCurl(CeedBasis basis, const CeedScalar **curl);
 CEED_EXTERN int CeedBasisDestroy(CeedBasis *basis);
 
 CEED_EXTERN int CeedGaussQuadrature(CeedInt Q, CeedScalar *q_ref_1d, CeedScalar *q_weight_1d);
 CEED_EXTERN int CeedLobattoQuadrature(CeedInt Q, CeedScalar *q_ref_1d, CeedScalar *q_weight_1d);
 CEED_EXTERN int CeedQRFactorization(Ceed ceed, CeedScalar *mat, CeedScalar *tau, CeedInt m, CeedInt n);
 CEED_EXTERN int CeedSymmetricSchurDecomposition(Ceed ceed, CeedScalar *mat, CeedScalar *lambda, CeedInt n);
 CEED_EXTERN int CeedSimultaneousDiagonalization(Ceed ceed, CeedScalar *mat_A, CeedScalar *mat_B, CeedScalar *x, CeedScalar *lambda, CeedInt n);
+CEED_EXTERN int CeedHouseholderApplyQ(CeedScalar *A, const CeedScalar *Q, const CeedScalar *tau, CeedTransposeMode t_mode, CeedInt m, CeedInt n,
+                                      CeedInt k, CeedInt row, CeedInt col);
+CEED_EXTERN int CeedMatrixMatrixMultiply(Ceed ceed, const CeedScalar *mat_A, const CeedScalar *mat_B, CeedScalar *mat_C, CeedInt m, CeedInt n,
+                                         CeedInt kk);
```

Comment:
Why the move? These aren't optimized and not intended for external users

Reply:
Yeah sorry about this and I'm happy to move them back, this is an artifact of familiarizing myself with the library. I was just trying to make sense of where these methods were being used and defined in `ceed-basis.c`, and thought they could just go alongside the other utility linear algebra methods which these methods which were declared in `ceed.h` as opposed to `backend.h`.

Reply:
Reverted in [063bea1](https://github.com/CEED/libCEED/pull/1156/commits/063bea108b882e83a5ae894d5a49e974101cbd30).